src/gsy_e/gsy_e_core/util.py

```diff
@@ -443,21 +475,39 @@ def get_market_maker_rate_from_config(next_market, default_value=None, time_slot
                 raise e
         return find_object_of_same_weekday_and_time(GlobalConfig.market_maker_rate,
                                                     time_slot)
-    else:
-        return GlobalConfig.market_maker_rate
+    return GlobalConfig.market_maker_rate
+
+
+def get_feed_in_tariff_rate_from_config(next_market, default_value=None, time_slot=None):
```

Comment:
`next_market` is of `BaseMarkt` type right?
```suggestion
def get_feed_in_tariff_rate_from_config(next_market: "BaseMarket", default_value=None, time_slot=None):
```

Reply:
I assume, you meant following,

```suggestion
def get_feed_in_tariff_rate_from_config(next_market: "MarketBase", default_value=None, time_slot=None):
```